java-checks/src/main/resources/org/sonar/l10n/java/rules/java/S2653_java.json

```diff
@@ -1,27 +1,16 @@
 {
   "title": "Web applications should not have a \"main\" method",
   "type": "VULNERABILITY",
-  "status": "ready",
+  "status": "deprecated",
```

Comment:
There is a ticket concerning this change: [SONARJAVA-3699](https://jira.sonarsource.com/browse/SONARJAVA-3699).